docs-devsite/auth.md

```diff
@@ -442,6 +442,8 @@ Returns a [UserCredential](./auth.usercredential.md#usercredential_interface) fr
 
 If sign-in succeeded, returns the signed in user. If sign-in was unsuccessful, fails with an error. If no redirect operation was called, returns `null`<!-- -->.
 
+This method is not available in Node.js.
```

Comment:
Might be worth marking initializeRecaptchaConfig as unavailable in Node too.. since you can't really use it without RecaptchaVerifier. Initializing the config itself will work, but we can't protect Auth flows with Recaptcha outside of a browser. Adding @renkelvin to confirm.

Reply:
Added.